gdal/frmts/ecw/lookup.py

```diff
@@ -1,5 +1,5 @@
 # ******************************************************************************
-#  $Id$
+#  $Id: lookup.py d7e898f6034a9c6c0f83cfbff921434dfc2082be 2018-04-18 23:07:52 +1000 Ben Elliston $
```

Comment:
it looks like it undoes at least part of Python cleanups that have been done. Will need more careful merging